storage/remote/store.go

```diff
@@ -0,0 +1,144 @@
+package remote
+
+import (
+	"context"
+	"fmt"
+	"github.com/onflow/flow-archive/api/archive"
+	"github.com/onflow/flow-archive/codec/zbor"
+	"github.com/onflow/flow-emulator/storage"
+	"github.com/onflow/flow-emulator/storage/sqlite"
+	exeState "github.com/onflow/flow-go/engine/execution/state"
+	"github.com/onflow/flow-go/fvm/state"
+	"github.com/onflow/flow-go/fvm/storage/snapshot"
+	"github.com/onflow/flow-go/ledger/common/pathfinder"
+	"github.com/onflow/flow-go/ledger/complete"
+	flowgo "github.com/onflow/flow-go/model/flow"
+	"github.com/pkg/errors"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+type Store struct {
+	*sqlite.Store
+	client archive.APIClient
```

Comment:
maybe we can add fork height here, then we can split lookups with height below

Reply:
maybe can need to commit a block to fork and makes things a bit easier.

Reply:
I tried implementing this, but it seems there's a problem with read data, there's some data that is requested in the after fork heights that wasn't yet saved locally, so I believe I can't make this assumption.